src/test/java/org/isf/visits/Tests.java

```diff
@@ -574,42 +574,42 @@ public void testVisitGetSet() throws Exception {
 	}
 
 	@Test
-	public void testVisitToStringSMS() throws Exception {
+	void testVisitToStringSMS() throws Exception {
 		int id = setupTestVisit(false);
 		Visit visit = visitManager.findVisit(id);
 		assertThat(visit.toStringSMS()).isEqualTo("08/09/10 00:00");
 	}
 
 	@Test
-	public void testVisitToString() throws Exception {
+	void testVisitToString() throws Exception {
 		int id = setupTestVisit(false);
 		Visit visit = visitManager.findVisit(id);
 		assertThat(visit).hasToString("TestDescription - testService - 08/09/10 - 00:00:00");
 	}
 
 	@Test
-	public void testVisitFormatDateTime() throws Exception {
+	void testVisitFormatDateTime() throws Exception {
 		int id = setupTestVisit(false);
 		Visit visit = visitManager.findVisit(id);
 		assertThat(visit.formatDateTime(visit.getDate())).isEqualTo("08/09/10 - 00:00:00");
 	}
 
 	@Test
-	public void testVisitFormatDateTimeSMS() throws Exception {
+	void testVisitFormatDateTimeSMS() throws Exception {
 		int id = setupTestVisit(false);
 		Visit visit = visitManager.findVisit(id);
 		assertThat(visit.formatDateTimeSMS(visit.getDate())).isEqualTo("08/09/10 00:00");
 	}
 
 	@Test
-	public void testVisitEquals() throws Exception {
+	void testVisitEquals() throws Exception {
 		int id = setupTestVisit(false);
 		Visit visit = visitManager.findVisit(id);
 
-		assertThat(visit).isEqualTo(visit);
 		assertThat(visit)
-				.isNotNull()
-				.isNotEqualTo("someString");
+			.isEqualTo(visit)
```

Comment:
also here

Reply:
Removed the line as the same test is made a few lines lower in the method.